Examples/DeveloperSPMExample/SPMExample/DeepLinkNavigator.swift

```diff
@@ -10,33 +10,42 @@ import AppcuesKit
 
 // NOTE: This deep link implementation should not be taken as an example of best practices. It's not particularly scalable,
 // it's coupled to the specific view controller hierarchy of the app, and uses segues which can cause runtime crashes.
-class DeepLinkNavigator {
+class DeepLinkNavigator: AppcuesNavigationDelegate {
```

Comment:
FYI I realized I changed the `DeeplinkNavigator` in the CocoapodsExample example for universal links, but not the SPM one. It'd be good to eventually reconcile them. I'm happy to do that when the time comes.